packages/react-focus/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -74,6 +75,12 @@ describe('FocusZone', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  isConformant({
+    Component: FocusZone,
+    displayName: 'FocusZone',
+    disabledTests: ['as-renders-fc', 'as-renders-react-class', 'as-passes-as-value', 'has-top-level-file'],
```

Comment:
Seems like this one should have a top-level file? Also curious what happened with the `as` tests.